pom.xml

```diff
@@ -229,7 +229,6 @@
     <profile>
       <id>format</id>
       <activation>
-        <activeByDefault>true</activeByDefault>
```

Comment:
I would suggest to either keep this line or add a comment to the activation property, stating that this profile is "effectively activated by default" to clearly signal that this profile is active for a local build.

Reply:
Comment added